site/styles/_carousel.scss

```diff
@@ -138,7 +138,7 @@ s72-carousel {
   }
 
   .availability-bg {
-    margin-bottom: 8px;
+    margin-bottom: 22px;
```

Comment:
Where's this magic number from?

Reply:
@PeculiarGoat To make the spacing look right, it has to be eyeballed. Paragraphs and h tags have extra spacing at the top and bottom which is part of the font. Even worse, the spacing scales with the font size. Other elements like buttons and the availability label do not have this extra spacing. I've added an example of what it looks like when every element has `margin-bottom: 8px`. You can see that the spacing is cramped and inconsistent.

![with 8px](https://user-images.githubusercontent.com/41402426/181651738-31452f59-27b0-4777-98c0-48100e21688d.png)

The spacing around the availability label should be greater than the rest of the elements. Here's the designs.

![designs](https://user-images.githubusercontent.com/41402426/181651859-9695a3d9-4862-49e1-b388-0ca77b144200.png)

Here's what what it looks like now. This synopsis has `margin-bottom: 14px`, availability label has `margin-bottom: 22px`

![With 14px](https://user-images.githubusercontent.com/41402426/181653172-18279dc3-c101-40be-9efa-7688ba42e1cb.png)

Of course there are permutations to consider. In the future, elements below the synopsis could have the extra spacing from fonts or they could not. I think it is better to have too much spacing for a few outliers. The alternative is for things to look cramped 99% of the time.

Reply:
The mobile view looks bit squashed up and has more padding below the label. Would be good if equal padding is added above the label. 
![Screen Shot 2022-07-29 at 11 44 08 AM](https://user-images.githubusercontent.com/13471681/181655262-0a5b594a-71b1-4aa1-9cbf-70b07871b895.png)

Reply:
Is 22px bottom margin used anywhere else on the site?

Maybe this isn't the place to rage about the lack of a design language.